egs/ami/s5/local/nnet3/run_tdnn.sh

```diff
@@ -6,7 +6,7 @@
 # At this script level we don't support not running on GPU, as it would be painfully slow.
 # If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
 # --num-threads 16 and --minibatch-size 128.
-set -e
+set -x
```

Comment:
I kept it as I am still debugging the script, but of course normal users don't require this verbosity. removing it.